DataBiosphere/Saturn-UI
# [WOR-57] Use Sam ToS status if possible, otherwise rely on existing ToS cloud function

Ticket: [WOR-57](https://broadworkbench.atlassian.net/browse/WOR-57)
Testing:
* against a fiab
  * spin up a fiab with ToS feature branches ([sam](https://github.com/broadinstitute/sam/pull/568), [firecloud-develop](https://github.com/broadinstitute/firecloud-develop/pull/2771), [orchestration](https://github.com/broadinstitute/firecloud-orchestration/pull/913))
  * sign in to Terra with users that have not registered at all and users that are registered but haven't accepted the current ToS version. Verified that user was prompted with ToS in both situations and that user was no longer prompted after accepting. 
* against dev
  * I also ran this branch against dev to make sure it would be safe to merge now before the other ToS branches were merged. Verified that I was not prompted to accept ToS as I have already accepted the ToS via existing Cloud Function

<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>
